src/app/_graphql/contributors.ts

```diff
@@ -0,0 +1,21 @@
+export const CONTRIBUTORS = `
+  query Contributors {
+    Contributors(limit: 10) {
+      docs {
+        id
+        name
+        }
+    }
+  }
+`
+
+export const CONTRIBUTOR = `
```

Comment:
eventually, yes. i moved to podcast episodes because at the time i was struggling with auth
will get back to this once podcasts is wrapped 

(should've stuck with this until finishing it shouldn't I? :\)